src/ble.c

```diff
@@ -0,0 +1,253 @@
+#include "fido.h"
```

Comment:
These newly introduced files are missing a licensing header and copyright information. :-)